app/views/gardens/index.html.erb

```diff
@@ -1,9 +1,40 @@
-<div class="gardens-index">
+<!-- <div class="gardens-index">
   <nav></nav>
   <div class="gardens-headline">
     <h1>select your garden</h1>
   </div>
   <div class="gardens">
-    <%= render @gardens %>
+  <%= render @gardens %>
+</div>
+</div> -->
+
+<% garden = @gardens.sample %>
+<%= cl_image_tag garden.photo.key, height: 300, width: 400, crop: :fill %>
+
+
+<% @gardens.each do |garden| %>
```

Comment:
@georgkloeck - Please delete this loop - the code you commented out (above) <%= render @gardens %> does the same thing as your loop.  Please remove your loop and just put the image tag from cloudinary cl_image_tag into the _garden partial I created. thx